src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -182,4 +183,19 @@ public static Location parseLocation(String location) throws ParseException {
         return new Location(trimmedLocation);
     }
 
+    /**
+     * Parses a {@code String location} into an {@code Location}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code location} is invalid.
```

Comment:
JavaDocs not updated here

Reply:
Also just a heads up, I just reviewed Jonas' PR, and I am worried there might be potential merge conflicts between your PR and his. Do check if there might be!

Reply:
I think would have conflict. Just resove it when we merging.